Update launcher version and test new fd framework

sajw12 opened the pull request.

sajw12 committed:
Update launcher version and test new fd framework

sajw12 committed:
change krb5 version

sajw12 committed:
update krb5 sha

sajw12 committed:
Update com.albiononline.AlbionOnline.appdata.xml

sajw12 commented:
edited: never mind my ramblings, that was me making a mistake. 

I've tried the build from the testing repo as above and it seems to install and run the game correctly using fd23.08 and krb5 1.21.2 so I think it can be reviewed and if you are happy merged into the main branch. @AsciiWolf @valentindavid

AsciiWolf commented on com.albiononline.AlbionOnline.appdata.xml:
```diff
@@ -12,10 +12,10 @@
   <screenshots>
     <screenshot>
-      <image type="source">https://albiononline.com/wp-content/uploads/2014/05/Albion-Online-2014-06-13-13-38-35-59.jpg</image>
+      <image type="source">https://cdn.cloudflare.steamstatic.com/steam/apps/761890/ss_ace9036fa9be3f15a81ebeb64c5843ec79b72be4.jpg?t=1704713171</image>
```
```suggestion
      <image type="source">https://cdn.cloudflare.steamstatic.com/steam/apps/761890/ss_ace9036fa9be3f15a81ebeb64c5843ec79b72be4.jpg</image>
```

AsciiWolf commented on com.albiononline.AlbionOnline.appdata.xml:
```diff
@@ -12,10 +12,10 @@
     </screenshot>
     <screenshot>
-      <image type="source">https://albiononline.com/wp-content/uploads/2014/08/castle.jpg</image>
+      <image type="source">https://cdn.cloudflare.steamstatic.com/steam/apps/761890/ss_c197aa77b6f65af34fd51292e9033da6960ee23f.jpg?t=1704713171</image>
```
```suggestion
      <image type="source">https://cdn.cloudflare.steamstatic.com/steam/apps/761890/ss_c197aa77b6f65af34fd51292e9033da6960ee23f.jpg</image>
```

AsciiWolf commented:
I sadly do not have push rights to this repo. Anyway, it is great that this Flatpak is finally being updated to a recent fd runtime. :-) Thanks a lot for working on this!

sajw12 committed:
Update com.albiononline.AlbionOnline.appdata.xml

Co-authored-by: AsciiWolf <mail@asciiwolf.com>

sajw12 committed:
Update com.albiononline.AlbionOnline.appdata.xml

Co-authored-by: AsciiWolf <mail@asciiwolf.com>

sajw12 committed:
Update com.albiononline.AlbionOnline.appdata.xml

updated component tag per https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#header

AsciiWolf commented:
bot, build

barthalion commented:
The appdata file is just completely broken right now. Please validate it with appstreamcli locally to see all errors.

sajw12 commented:
bot, build

sajw12 committed:
Update com.albiononline.AlbionOnline.appdata.xml

wrote new metadata file to compy with freedesktop standard

sajw12 commented:
bot, build
